collections/Laud_Misc/MS_Laud_Misc_132.xml

```diff
@@ -74,7 +116,108 @@
+                     </msItem>
+                  </msContents>
+                  <physDesc>
+                     <objectDesc form="sheet">
+                        <supportDesc material="perg">
+                           
+                           <support>parchment</support>
+                           <extent>
+                              <dimensions unit="mm" type="fragment">
+                                 <height>283</height>
+                                 <width>301</width>
+                              </dimensions>
+                           </extent>
+                        </supportDesc>
+                        <layoutDesc>
+                           <layout columns="1">Long lines</layout>
+                        </layoutDesc>
+                     </objectDesc>
+                     <handDesc>
+                        <handNote script="cursiva">Cursive (bastarda)</handNote>
+                     </handDesc>
+                     <decoDesc>
+                        <decoNote type="plainInit">Initial in the ink of the text.</decoNote>
+                     </decoDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" when="1458" cert="high">1458 (c.)</origDate>
+                        <origPlace><country key="place_7000084">Germany</country></origPlace>
+                     </origin>
+                     <provenance></provenance>
+                  </history>
+               </msPart>
+               <msPart xml:id="MS_Laud_Misc_132_endleaves-2" type="fragment">
+                  <msIdentifier>
+                     <altIdentifier type="partial">
+                        <idno type="part">MS. Laud Misc. 132 - endleaves (fols. 75-6)</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+                  <msContents>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem xml:id="MS_Laud_Misc_132_endleaves-2-item1">
+                        <title type="desc" key="work_13355">Missal (fragment)</title>
+                        <note>Tract 'Domine audivi ...' with verses 'In medio duorum animalium ...', 'Operuit celos ...', 'Deus a libano veniet ...', 'In eo dum conturbata ...' (cf. Cantus Index g02680); prayer 'Deus qui peccati ueteris hereditariam ...'; reading from Exodus; tract 'Eripe me domine ...' with verses 'Qui cogitauerunt malicias ...', 'Accuerunt linguas ...', 'Custodi me domine ...', 'Qui cogitauerunt subplantare ...', 'Et funes extenderunt ...' , 'Dixi domino ...', 'Domine domine uirtus salutis ...', 'Ne tradas me a desiderio ...', 'Caput circuitis ...', 'Veruntamen iusti ...' (Cantus Index g02431, 2431a-j); passion reading with voices (c, s, t).</note>
```

Comment:
These are for [Good Friday (Feria 6 in Parasceve)](http://cantusindex.org/feast/07066000).